xla/backends/gpu/runtime/command_buffer_cmd.h

```diff
@@ -202,6 +202,10 @@ class CommandBufferCmd {
     // An external state manager that gives efficient access to per-device state
     // to commands without a need to add expensive synchronization.
     StateManager& state;
+
+    // A reference to the vector of device memory bases that are previously
```

Comment:
Why it's needed at all? Command sequence only needs to know if allocation at given index changed or not, concrete address should not matter. And this information should stay only at CommandBufferCmdSequence level, individual commands should not care about that.

Reply:
Fixed